ratelimit/ratelimit.go

```diff
@@ -456,7 +457,13 @@ func Headers(s *Settings, retryAfter int) http.Header {
 	}
 }
 
-func getHashedKey(clearText string) string {
-	h := sha256.Sum256([]byte(clearText))
-	return hex.EncodeToString(h[:])
+func getKey(group, text string) string {
+	return getKeyImpl(group, text, md5.New()) // #nosec
```

Comment:
It should have minimal prefix in order to put ratelimit keys into separate namespace as there may be other key namespaces in the future.